Composer/packages/client/src/utils/publishStatusPollingUpdater.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import httpClient from './httpUtil';
```

Comment:
It feels like you could leverage the useIntervalHook to accomplish the polling instead of building a class component for polling. Can you check the component BotRuntimeStatus.tsx?

Reply:
The reason for using polling class is that we can not only clean up actions when a component destroyed, but also can stop the interval manually.  And the polling updater is a list, we can manage the updater object easier.

Reply:
I understand the reason to use polling but why not just use the useInterval hook to accomplish it